api.md

```diff
@@ -1229,6 +1230,39 @@ For a table with more than one space dimension:
 SELECT set_number_partitions('conditions', 2, 'device_id');
 ```
 
+
+---
+
+## set_replication_factor() [](set_replication_factor)
+Sets the replication factor of a distributed hypertable to the given value. 
+The setting will affect only new chunks. 
+Newly created chunks of the hypertable will be replicated to different data nodes,
+so the amount of replicas is the same as the replication factor.
```

Comment:
Slightly unclear. Suggestion:

> Changing the replication factor will not change the number of replicas for existing chunks, but any chunks created after changing the replication factor will use the new value of the replication factor.

Reply:
Thanks for noting and suggestion. I made an improvement, which I hope helps.